drivers/sensor/lsm303dlhc_accel/Kconfig

```diff
@@ -0,0 +1,79 @@
+# Kconfig - LSM303DLHC Three Axis Accelerometer configuration options
+
+#
+# Copyright (c) 2018 Philémon Jaermann
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig LSM303DLHC_ACCEL
+	bool
+	prompt "LSM303DLHC Three Axis Accelerometer"
```

Comment:
You can combine lines 10-11 into one:
```
bool "LSM303DLHC Three Axis Accelerometer"
```

Please do this throughout the whole file